ondewo/nlu/agent.proto

```diff
@@ -413,6 +413,10 @@ message ExportAgentRequest {
     // Note: The URI must start with
     // "gs://". If left unspecified, the serialized agent is returned inline.
     string agent_uri = 2;
+
+    // Optional. The compression level of the zip file created.
+    // Valid range 1-9; other values are replaced by the default value 1
```

Comment:
oh, so, we are not raising an error but amending the input value?

Reply:
I think you're right, raising an error would be better. I'll make a change.